README.md

```diff
@@ -16,8 +16,8 @@ Add `open_telemetry_decorator` to your list of dependencies in `mix.exs`. We inc
 ```elixir
 def deps do
   [
-    {:open_telemetry_decorator, "~> 0.5.3"},
-    {:opentelemetry, "~> 0.4.0"}
+    {:open_telemetry_decorator, "~> 1.0.0-rc.3"},
```

Comment:
Why a `rc` version? Is it safe to use?

Reply:
Probably means release candidate. IMO, if we do tests in all environments and none problem raises, it's safe. All these opentelemetry forks that we have and use are much behind the current state and we should fix that

Reply:
Totally agree, I was just afraid of `rc` means `running changes` hahaha